pkg/cluster/cluster.go

```diff
@@ -760,6 +763,30 @@ func (c *Cluster) initHumanUsers() error {
 	return nil
 }
 
+func (c *Cluster) initHumanUsers() error {
+
+	for _, postgresSuperuserTeam := range c.OpConfig.PostgresSuperuserTeams {
+		err := c.initTeamMembers(postgresSuperuserTeam, true /*isPostgresSuperuserTeam*/)
+		if err != nil {
+			return fmt.Errorf("Cannot create a team %q of Postgres superusers: %v", postgresSuperuserTeam, err)
+		}
+	}
+
+	for _, postgresSuperuserTeam := range c.OpConfig.PostgresSuperuserTeams {
```

Comment:
is there no "in" operator? for String array?

Reply:
[apparently no due to the lack of generics in the language](https://stackoverflow.com/questions/10485743/contains-method-for-a-slice)

Reply:
![](https://www.reactiongifs.us/wp-content/uploads/2014/01/ba_dum_tss_pirates_band_of_misfits.gif)

Reply:
@zerg-junior but why this part can't be merged with the previous loop?